services/linetracking.md

```diff
@@ -0,0 +1,31 @@
+# Line tracking
+
+A sensor that detects light and dark surfaces, typically used for line following robots.
+
+    identifier: 0x126c4cb2
+    extends: _sensor
+
+## Registers
+
+    ro brightness: u8 / @ reading
+
+Reports the detected brightness. It may be a digital value or, for some sensor, analog value.
```

Comment:
Let's go for u16 - most ADCs are more than 8 bit.

Also what do you mean digital or analog value?

Reply:
yes. most sensor reports high/low but some have more sensors and are better suited for PID control